packages/frontend/core/src/modules/collection/services/collection.ts

```
@@ -19,6 +19,11 @@ export class CollectionService extends Service {
     },
   });
 
+  readonly collectionDataReady$ = LiveData.from(
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)